workflows/microbiome/mgnify-amplicon-pipeline-v5-complete/README.md

```diff
@@ -0,0 +1,17 @@
+# MGnify's amplicon pipeline (v5.0)
```

Comment:
I assume this is the main entry point for the user (or can there be a README for all workflows ? - then this would probably be better !)
Can we add: 
* What it this? Can you add a few lines of explanation (what is MGnify, what is the amplicon pipeline, what is the advantage of this port, what do the individual workflows do ... )
* How can one test the workflow and see that the results are the same. 
* Explain that the classification part is not the same due to mapseq (show example)